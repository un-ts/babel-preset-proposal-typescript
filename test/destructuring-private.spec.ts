import _ from './destructuring-private.js'
import { execute } from './helpers.js'

const proposal = 'destructuring-private'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot(
    `"Unexpected token ';'"`,
  )
  expect(_(1, 0)).toEqual(true)
  expect(_(0, 1)).toEqual(false)
})
