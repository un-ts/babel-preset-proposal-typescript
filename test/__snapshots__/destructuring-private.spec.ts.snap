// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`destructuring-private: "Unexpected token ';'" 1`] = `"Unexpected token ';'"`;
